### ISSUE TITLE
KL divergence formula

### ISSUE THREAD

**Issue by tvoesolncee** [snousias/fast-mesh-denoising#1]
Hello!
I don`t actually understand the loss function in your code for CVAE.
Where do the values 0.15 and 1e-8 come from in formulation of Kullback-Leibler divergence?
And why is there "+" before 1 in formula instead of "-"?

```
KL_Div = tf.reduce_mean(0.15 * (1 - tf.math.log(tf.square(std) + 1e-8) + tf.square(mean) + tf.square(std)))
```
